### PR TITLE
Added Gitea icon

### PR DESCRIPTION
**Pull request by JoaoPedroMoraes**
**Issue:** #1711 

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
From the official SVG I was able to create a monocolor version. https://gitea.com/

the svg logo was removed from header https://github.com/go-gitea/gitea/blob/master/assets/logo.svg

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -1120,6 +1120,11 @@
             "hex": "F05032",
             "source": "http://git-scm.com/downloads/logos"
         },
+        {
+            "title": "Gitea",
+            "hex": "609926",
+            "source": "https://gitea.com/"
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: Where on this page can you find the SVG you used?

If it's not there I think the link provided by @PeterShaggyNoble in #1711 is a better alternative. Agreed?

```suggestion
            "source": "https://github.com/go-gitea/gitea/tree/master/assets"
```

Comment (PeterShaggyNoble) [reply _data/simple-icons.json]: Agreed on the source URL, as it's the source of both the original SVG and the chosen colour.

**Comment by JoaoPedroMoraes** [comment]
svg fixed and link updated :+1:

**Comment by ericcornelissen** [comment]
Thanks for the update and thanks for working on this @JoaoPedroMoraes!

**Pull request merged**